src/User32.Desktop/User32+PAINTSTRUCT.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="PAINTSTRUCT"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct PAINTSTRUCT
+        {
+            public bool fErase;
+            public bool fIncUpdate;
+            public bool fRestore;
+            public IntPtr hdc;
+            public RECT rcPaint;
+
+            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 32)]
```

Comment:
Using `MarshalAs` on struct fields makes the fields non-blittable, so that we can't pass pointers to them in p/invoke signatures. 
Instead, use this:

```csharp
public fixed byte rgbReserved[32];
```